conf/eddie.config

```diff
@@ -20,7 +20,7 @@ process {
             return null
         }
         // total bytes requested, divided by slots ⇒ bytes per slot
-        def bytesPerSlot = task.memory.bytes / task.cpus
+        def bytesPerSlot = task.memory.bytes
```

Comment:
I can't do suggestion outside of the diff now itseff, but wouldn't this be simpler with just a

```groovy
    return "-l h_vmem=${task.memory.toGiga()}G"
```

? With the old naming retained, it seems like it's something that could easily lead to confusion.

Also, I assume this works but I don't think `bytes` is listed in the described interface for [MemoryUnit](https://www.nextflow.io/docs/latest/reference/stdlib-types.html#memoryunit).